process_tf.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python
+
+from argparse import ArgumentParser
+from collections import defaultdict
+from os import listdir, mkdir
+from os.path import isfile, join, isdir
+
+dbs_by_owner = defaultdict(list)
+
+
+def get_args():
+    parser = ArgumentParser()
+    parser.add_argument('-s', '--source-directory', type=str, help="Name of the source directory with .tf files")
+    parser.add_argument('-d', '--dest-directory', type=str, help="Name of the dest directory for new .tf files")
+
+    return parser.parse_args()
+
+
+def get_brackets_sum(line):
+    return line.count('{') - line.count('}')
+
+
+def iterate_til_closing_bracket(lines, index):
+    between_brackets = []
+    b_sum = 0
+    for i in range(index, len(lines)):
+        line = lines[i]
+        between_brackets.append(line)
+        b_sum += get_brackets_sum(line)
+        if b_sum == 0:
+            return between_brackets, i
+    raise Exception("Can not parse line:\n%s", lines[index])
+
+
+def prepare_db(db_lines, cluster_resource_type, cluster_resource_name):
+    prefix_length = db_lines[0].find('database')
+    db_name = None
+    db_owner = None
+    new_db_lines = []
+    for i in range(len(db_lines)):
+        db_lines[i] = db_lines[i][prefix_length:]
+    i = 0
+
+    while i < len(db_lines):
+        splitted_line = db_lines[i].split()
+        if len(splitted_line) == 3 and splitted_line[0] == 'name' and splitted_line[1] == '=':
+            db_name = splitted_line[2]
+            if db_name[0] == db_name[-1] == '"' or db_name[0] == db_name[-1] == "'":
+                db_name = db_name[1:-1]
+        if len(splitted_line) == 3 and splitted_line[0] == 'owner' and splitted_line[1] == '=':
+            db_owner = splitted_line[2]
+            if db_owner[0] == db_owner[-1] == '"' or db_owner[0] == db_owner[-1] == "'":
+                db_owner = db_owner[1:-1]
+        if len(splitted_line) == 2 and splitted_line[0] == 'extension' and splitted_line[1] == '{':
+            extensions, i = iterate_til_closing_bracket(db_lines, i)
+            new_db_lines += extensions
+            i += 1
+            continue
+        new_db_lines.append(db_lines[i])
+        i += 1
+    if not db_name or not db_owner:
+        raise Exception("Can not find out db name/owner in %s %s", cluster_resource_type, cluster_resource_name)
+    db_lines[0] = f'resource "yandex_mdb_postgresql_database" "{cluster_resource_name}-{db_name}" {{'
+    cluster_id_line = f'  cluster_id = {cluster_resource_type}.{cluster_resource_name}.id'
+    db_lines = [db_lines[0], cluster_id_line] + db_lines[1:]
+    dbs_by_owner[cluster_resource_name, db_owner].append(db_name)
+    return db_lines
+
+
+def prepare_permissions(permissions, dbs_to_exclude):
```

Comment:
I think it would be useful to add comments. What exactly makes this method?
Well, I know now it is remove dbs where the user is owner but we'll forget it :)